controllers/subscriptions.go

```diff
@@ -346,191 +334,90 @@ func ApproveInstallPlanForCsv(cli client.Client, csvName string) error {
 	return finalError
 }
 
-// GetSubscriptions returns all required Subscriptions
-func GetSubscriptions() []*operatorv1alpha1.Subscription {
+func GetOlmPkgRecord() []*OlmPkgRecord {
```

Comment:
I don't think there is a need here to make this a public function
```suggestion
func getOlmRecords() []*OlmPkgRecord {
```

Reply:
Same as above.